Frontend/ui-driver/src/Flow.purs

```diff
@@ -352,6 +352,9 @@ getDriverInfoFlow = do
 
       let dbClientVersion = getDriverInfoResp.clientVersion
       let dbBundleVersion = getDriverInfoResp.bundleVersion
+      let maskedDeviceToken = getDriverInfoResp.maskedDeviceToken
+      when (any (_ == maskedDeviceToken) [Just "__f...led", Just "..."]) do
```

Comment:
`__f...led` `...` these values are from backend, Is it by default?
Can you please explain this

Reply:
These values from Backend and these two cases are failed cases